lib/rspec/core/configuration_options.rb

```diff
@@ -13,6 +13,10 @@ def initialize(args)
         organize_options
       end
 
+      def command_line_and_env_args
```

Comment:
It's not clear to me why we need a method containing both command line and env args but not args from the dot files.  Can you explain?